src/features/token/types/CorporateTokenTransaction.ts

```diff
@@ -32,7 +32,7 @@ export interface CorporateTokenProgramSales {
   qty: number
   finalPrice: number
   notes: string
-  attendees: Attendee[]
+  attendees: BodyAttendee[]
```

Comment:
```suggestion
  attendees: CorporateAttendee[]
```
kayanya lebih enak gini